recipes/SLURP/direct/hparams/train_with_wav2vec2.yaml

```diff
@@ -0,0 +1,183 @@
+# ############################################################################
+# Model: Direct SLU
+# Encoder: Pre-trained wav2vec2 -> LSTM
+# Decoder: GRU + beamsearch
+# Tokens: BPE with unigram
+# losses: NLL
+# Training: SLURP
+# Authors:  Loren Lugosch, Mirco Ravanelli 2020 Boumadane Abdelmoumene, AbdelWahab Heba, Yingzhi Wang 2021
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+# The SLURP dataset will be automatically downloaded in the specified data_folder
+data_folder: !PLACEHOLDER # e.g, /localscratch/SLURP
+data_folder_rirs: !ref <data_folder>
+train_splits: ["train_synthetic", "train_real"]
+csv_train: !ref <output_folder>/train-type=direct.csv
+csv_valid: !ref <output_folder>/devel-type=direct.csv
+csv_test: !ref <output_folder>/test-type=direct.csv
+tokenizer_file: https://www.dropbox.com/s/w72vtnf273yjwu4/SLURP_58_unigram.model?dl=1
+skip_prep: False
+
+
+# URL for the wav2vec2 model, you can change to benchmark diffrenet models
+wav2vec2_hub: "facebook/hubert-base-ls960"
+
+# Training parameters
+number_of_epochs: 35
+batch_size: 2
+lr: 0.0003
+lr_wav2vec2: 0.00001
+token_type: unigram # ["unigram", "bpe", "char"]
+sorting: random
+ckpt_interval_minutes: 15 # save checkpoint every N min
+
+#freeze all wav2vec2
+freeze_wav2vec2: False
+
+#set to true to freeze the CONV part of the wav2vec2 model
+freeze_wav2vec2_conv: True
```

Comment:
Do you see any difference when freezing the CNN conv ?

Reply:
yes with entire fine-tuning I got 87.44% on intent classification and 75.06% on slot filling, which is a little little bit lower